smart-events-user-dashboard/src/Login.tsx

```diff
@@ -1,34 +1,54 @@
 import * as React from 'react';
+import './App.css'
 import {
-  Container, 
+  Box,
   Avatar, 
   TextField, 
-  Button 
+  Button,
+  Card,
+  CardMedia, 
+  Typography,
 } from '@mui/material';
 import '@fontsource/roboto/300.css';
 import '@fontsource/roboto/400.css';
 import '@fontsource/roboto/500.css';
 import '@fontsource/roboto/700.css';
+import { 
+  flexbox,
+  spacing,
+} from '@mui/system';
 
 function Login() {
-
   return (
-    <div className="Login">
-      <Container>
-        <Avatar 
-          alt="Yellow Jacket Logo"
-          src="/src/assets/yellow-jacket-logo.png"
-          variant="square"
+    <Card
+      sx={{
+        p: 5,
+        height: 350,
+        display: 'flex',
+        flexDirection: 'column',
```

Comment:
Again, great use of flexbox.

Reply:
Note that this here may actually be an appropriate use of `sx` as it's overriding a specific instance of flex orientation.